Sources/Examples/HelloWorld/Client/HelloWorldClient.swift

```diff
@@ -25,6 +25,9 @@ struct HelloWorld: AsyncParsableCommand {
   @Option(help: "The port to connect to")
   var port: Int = 1234
 
+  @Flag(help: "Use local vsock socket")
```

Comment:
I don't think we should include vsock in the examples: this one has an accompanying tutorial and we should keep the code in line with that. We should keep them as simple as possible: most users won't use vsock so we shouldn't expose them to it here.

Reply:
That's completely fine. I assumed this was the case and had them here purely for the purposes of the draft PR to show them in action.

Removed.